Makefile

```diff
@@ -40,7 +41,8 @@ test: build-test-container test-in-container test-doc-examples
 
 test-in-container:
 	@# use it like this: `make test-in-container TEST_TARGET=tests/integration/test_utils.py`
-	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v $(CURDIR):/src -v $(CURDIR)/pytest-container.ini:/src/pytest.ini $(TEST_IMAGE_NAME) make exec-test TEST_TARGET=$(TEST_TARGET)
+	sed -e s#"${HOME}"#/root#g ${HOME}/.kube/config > $(kubedir)/config
```

Comment:
oh boy, do you really want to change files in my $HOME?

Reply:
@TomasTomecek could this affect the source file (the one in home)? why is this not safe?

Reply:
Oh, my bad! I thought it's `-i $HOME...`. Should work just fine.